lib/appscale_tools.py

```diff
@@ -47,6 +46,13 @@ class AppScaleTools(object):
   # down.
   SLEEP_TIME = 5
 
+  # The initial amount of time we should sleep when waiting for UserAppServer
+  # metadata to change state.
+  STARTING_SLEEP_TIME = 1
```

Comment:
It was used initially but not anymore and I forgot to clean up afterwards. Will remove that!